contrib/postgresql/template.py

```diff
@@ -0,0 +1,106 @@
+pkgname = "postgresql"
+pkgver = "16.2"
+pkgrel = 0
+build_style = "configure"
+configure_args = [
+    # TODO: review path choices - alpine puts pkgname in a few of these
+    "--bindir=/usr/bin",
+    "--libdir=/usr/lib",
+    "--includedir=/usr/include",
+    "--datadir=/usr/share",
+    "--disable-rpath",  # TODO: alpine does this but the docs say not to. should we?
+    "--with-system-tzdata=/usr/share/zoneinfo",
+]
+make_cmd = "gmake"
+makedepends = [
+    "gmake",
+    "pkgconf",
+    "icu-devel",
+    "readline-devel",
+    "zlib-devel",
+    "linux-headers",
+]
+depends = ["tzdata"]
+pkgdesc = "Sophisticated object-relational DBMS"
+maintainer = "mia <mia@mia.jetzt>"
+license = "PostgreSQL"
+url = "https://www.postgresql.org"
+source = [
+    f"https://ftp.postgresql.org/pub/source/v{pkgver}/postgresql-{pkgver}.tar.bz2"
+]
+sha256 = ["446e88294dbc2c9085ab4b7061a646fa604b4bec03521d5ea671c2e5ad9b2952"]
+# TODO: checks rely on libpq already being installed
+# could maybe makedepend on libpq once it's in the repos?
+# note: alpine doesn't do checks either
```

Comment:
this just means to install for tests, i.e. running meson install as part of check and running a check on that tmpdir. it's not impossible, but let's leave this for later and decide then (i'm not sure if there is a minimised set- the full testsuite is unreasonable and takes forever iirc, since it's a whole database thing)